aspnet/fundamentals/localization.rst

```diff
@@ -204,6 +204,9 @@ CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+ASP.NET Core use ``.AspNetCore.Culture`` cookie by default. You have to use format ``c=%LANGCODE%|uic=%LANGCODE%``, where ``c`` is CultureInfo and ``uic`` is CultureUIInfo, for instance:
+c='en-UK'|uic='en-US'
+The ``uic`` keypair is optional.
```

Comment:
It's my understanding that you can provide either `c` or `uic` and it's equally valid and will just populate the value of one with the other.